.gitignore

```
@@ -0,0 +1,2 @@
+node_modules
```

- [ ] node_modules is a folder, kindly add a `/` according to practises. 😄 
```suggestion
node_modules/
```